src/Sylius/Bundle/CoreBundle/Distributor/IntegerDistributor.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Distributor;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class IntegerDistributor implements IntegerDistributorInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function distribute($taxItems, $taxAmount)
```

Comment:
This is integer distributor, not specific to taxes, so this should be: $numberOfTargets, $amount or something like that.